### ISSUE TITLE
Add missing skip links

### ISSUE THREAD

**Issue by ruthmoog** [ruthmoog/portfolio#2]
A Skip link was not found or points to a non-existent id attribute. Skip links are required to allow screen reader users to bypass navigation links and go directly to the main content. Include skip navigation links. [WCAG 2.1 (Level A) - 2.4.1 ](https://www.w3.org/WAI/WCAG21/quickref/#bypass-blocks)

https://www.alumnionlineservices.com/accessibility/scanner/

**Comment by ruthmoog** [comment]
- [ ] commit:  [Add a skip link for keyboard users ](https://github.com/ruthmoog/portfolio/commit/7420356c08409d217d82d2d31997422dbfa8f909)
